src/bin/ion/commands/inspect.rs

```diff
@@ -361,6 +402,71 @@ impl<'a, 'b> IonInspector<'a, 'b> {
         Ok(())
     }
 
+    fn inspect_end_of_stream(&mut self, position: usize) -> Result<()> {
+        let mut empty_bytes = BytesFormatter::new(BYTES_PER_ROW, vec![]);
+        self.newline()?;
+        self.write_offset_length_and_bytes(0, position, "", &mut empty_bytes)?;
+        self.write_with_style(comment_style(), "// End of stream")
+    }
+
+    fn inspect_eexp(&mut self, depth: usize, eexp: EExpression<AnyEncoding>) -> Result<()> {
+        let mut formatter = BytesFormatter::new(
+            BYTES_PER_ROW,
+            vec![
+                // TODO: Add methods to EExpression that allow nuanced access to its encoded spans
+                // TODO: Length-prefixed and multibyte e-expression addresses
+                IonBytes::new(BytesKind::MacroId, &eexp.span().bytes()[0..1]),
+            ],
```

Comment:
🗺️ For now we assume that the e-expression's ID is just the first byte of the encoding. We need to add a method here to get the correct byte slice more reliably.